packages/router/src/bindings/fastify/index.ts

```diff
@@ -72,7 +72,7 @@ export const bindFastify = () =>
         }
         try {
           const senderTx = await prepareCancel({ senderChainId, user, transactionId });
-          const result = await contractWriter.cancel(
+          const result = await contractWriter.cancelTransactionManager(
```

Comment:
what if they have a contract based router?

Reply:
Not handled yet, can deal with this later. Trying to reduce scope still.